app/assets/stylesheets/pageflow/themes/_base.scss

```diff
@@ -192,3 +200,56 @@ section.delayed_text_fade_in_short {
+.page .text_page .scroller > div .contentInnerWrapper .contentText, .page .text_page .scroller > div .contentInnerWrapper .contentText, .js .page .text_page .scroller > div .contentInnerWrapper .contentText, .js .page .text_page .scroller > div .contentInnerWrapper .contentText, .js .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText, .js .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText, .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText, .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText {
+  @include desktop {
+    padding-top: 10em;
+  }
+}
+
+// Title size
+.page .text_page .scroller > div .contentInnerWrapper .contentText h3, .page .text_page .scroller > div .contentInnerWrapper .contentText h3, .js .page .text_page .scroller > div .contentInnerWrapper .contentText h3, .js .page .text_page .scroller > div .contentInnerWrapper .contentText h3, .js .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText h3, .js .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText h3, .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText h3, .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText h3 {
+  @include desktop {
+    font-size: 3em;
+    line-height: 1.2em;
+    padding-bottom: 1em;
+  }
+}
+
+// Text size
+.page .text_page .scroller > div .contentInnerWrapper .contentText p, .page .text_page .scroller > div .contentInnerWrapper .contentText p, .js .page .text_page .scroller > div .contentInnerWrapper .contentText p, .js .page .text_page .scroller > div .contentInnerWrapper .contentText p, .js .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText p, .js .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText p, .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText p, .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText p {
+  @include desktop {
+    font-size: 1.6em;
+  }
+}
+
+// Make text wider and scroll to the next one later
+.page .text_page .scroller > div .contentInnerWrapper .contentText, .page .text_page .scroller > div .contentInnerWrapper .contentText, .js .page .text_page .scroller > div .contentInnerWrapper .contentText, .js .page .text_page .scroller > div .contentInnerWrapper .contentText, .js .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText, .js .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText, .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText, .page:first-child .text_page .scroller > div .contentInnerWrapper .contentText {
+  @include desktop {
+    max-width: 700px;
+  }
+}
+
+.page .contentText  h1, .page .contentText  h2 span.subtitle, .page .contentText  h2 span.tagline, .page .contentText  h3, .page .contentText  p {
```

Comment:
![schermafbeelding 2017-03-03 om 17 52 14](https://cloud.githubusercontent.com/assets/14680/23560317/461b62a0-003a-11e7-91a3-09985b07218d.png)

@studiorabota kan het zijn dat door deze regel de weergave bij VK nu opeens stuk is? body tekst is veel te breed. ik haal deze regel nu sowieso weg